flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/pattern/conditions/IterativeCondition.java

```diff
@@ -84,7 +87,7 @@
 	/**
 	 * The context used when evaluating the {@link IterativeCondition condition}.
 	 */
-	public interface Context<T> {
+	public interface Context<T> extends TimerContext {
```

Comment:
Annotate this as `@PublicEvolving` as well.

Reply:
Isn't the `@PublicEvolving` transitive?  We don't do it for `org.apache.flink.streaming.api.functions.ProcessFunction.Context`, `org.apache.flink.streaming.api.functions.ProcessFunction.OnTimerContext`. I thought we use separate annotations on members only if we want to limit the guarantees.